firefox-ios/Client/metrics.yaml

```diff
@@ -2712,6 +2712,25 @@ preferences:
       - fx-ios-data-stewards@mozilla.com
     expires: "2024-06-01"
 
+# Private Browsing
+private_browsing:
+  data_clearance_icon_tapped:
+    type: event
+    description: |
+      Records when the private browsing button is tapped.
+    extra_keys:
+      did_confirm:
+       type: string
+       description: |
+         Whether the user confirmed the reset action via the confirmation dialog.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/18078
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/18080
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
+    expires: "2024-07-06"
```

Comment:
Let's align this with the rest so we can renew at the same timem-> 2024-06-01

Reply:
There are existing analytics that use `"2024-07-06"`, so Andy and I opted for the latest date. Should we use ` 2024-06-01` instead?

Reply:
@afurlan-firefox

Reply:
Ok that's fine, leave as is.